pmg/templates/admin/my_edit.html

```diff
@@ -12,6 +12,40 @@
     </div>
     {% endif %}
 
+    {% if admin_view.name == 'Users' %}
+
+    <button type="button" class="btn btn-danger btn-sm pull-right" data-toggle="modal" data-target="#password-reset-modal">
```

Comment:
Can we put all of this into a file included using {% include %} -- would make it a bit cleaner.